sdks/python/apache_beam/transforms/stats_test.py

```diff
@@ -496,6 +483,51 @@ def test_approximate_unique_globally_by_error_with_skewed_data(self):
           equal_to([True]),
           label='assert:globally_by_error_with_skewed_data')
 
+  def test_approximate_unique_combine_fn_by_nondeterministic_coder(self):
+    # test if the combiner throws an error with a nondeterministic coder.
+    sample_size = 30
+    coder = coders.Base64PickleCoder()
+
+    with self.assertRaises(ValueError) as e:
+      _ = ApproximateUniqueCombineFn(sample_size, coder)
+
+    self.assertRegex(
+        e.exception.args[0],
+        'The key coder "Base64PickleCoder" '
+        'for ApproximateUniqueCombineFn is not deterministic.')
+
+  def test_approximate_unique_combine_fn_add_values(self):
```

Comment:
nit: you could call this: test_approximate_unique_combine_fn_adds_values_correctly instead of adding a comment.

Reply:
done.